core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/dot-edit-content-category-field.component.scss

```diff
@@ -0,0 +1,34 @@
+@use "variables" as *;
+
+.dot-category-field__wrapper {
+    .dot-category-field__categories,
+    .dot-category-field__select {
+        border: $field-border-size solid $color-palette-gray-400;
+        display: flex;
+        flex-wrap: wrap;
+        align-items: center;
+    }
+
+    .dot-category-field__categories {
+        padding: $spacing-1;
+        gap: $spacing-1;
+        border-radius: $border-radius-md $border-radius-md 0 0;
+    }
+
+    .dot-category-field__select {
+        height: $field-height-md;
+        border-radius: $border-radius-md;
+        padding: 0 $spacing-0;
+        justify-content: flex-end;
+    }
+
+    &.has-categories {
+        .dot-category-field__select {
+            border-radius: 0 0 $border-radius-md $border-radius-md;
+        }
+
+        .dot-category-field__categories {
+            border-bottom: none;
+        }
+    }
```

Comment:
Nesting no needed, BEM does it implicitely.

Reply:
Improved